pkg/grafana/grafana.go

```diff
@@ -112,11 +105,6 @@ func UpdateOrganization(ctx context.Context, grafanaAPI *client.GrafanaHTTPAPI,
 		return nil
 	}
 
-	err = assertNameIsAvailable(ctx, grafanaAPI, organization)
-	if err != nil {
-		return errors.WithStack(err)
-	}
-
```

Comment:
Why is this gone ?

Reply:
It's moved up top before the create or update

Reply:
https://github.com/giantswarm/observability-operator/pull/233/files#diff-c9f8782f2e734927815792e50118cb136743e8fc9b086ce94ed7e45514451214L76

Reply:
oh right, missed that